### PR TITLE
Wimglenn/fixups

### PR DESCRIPTION
**Pull request by nicoddemus**
Reopen #26 to see if Travis executes properly.

### CODE REVIEW

**Comment by wimglenn** [comment]
@nicoddemus   Hmm.  I don't think `run.py` should be executed for pull requests.  Only the test suite.

File: .travis.yml [hunk 1]
```diff
@@ -10,18 +9,23 @@ before_script:
...
-  - pip install -r requirements.txt colorama
+  - pip install -r requirements.txt -U pytest==$PYTEST_VERSION
+
+jobs:
+  include:
+    - env: RUN_COMMAND="python run.py" PYTEST_VERSION=3.7.3 PLUGINCOMPAT_SITE=http://plugincompat.herokuapp.com
+      python: '2.7'
+    - env: RUN_COMMAND="python run.py" PYTEST_VERSION=3.7.3 PLUGINCOMPAT_SITE=http://plugincompat.herokuapp.com
+      python: '3.6'
+    - env: RUN_COMMAND="pytest test_web.py"
```
Comment (nicoddemus) [inline .travis.yml]: @wimglenn this is the main change I did, I think it makes sense to always run `test_web.py` and the `run.py` job, even on PRs.

**Comment by nicoddemus** [comment]
> I don't think run.py should be executed for pull requests. Only the test suite.

I disagree, this way we can get errors introduced in `run.py` too. PRs won't be able to actually post the results because they don't have the credentials for it.

File: .travis.yml (hunk 1 above)
Comment (wimglenn) [reply .travis.yml]: But then run.py will execute even if it has bad changes.  Also I thought the secret is not supposed to be available in PRs?

**Comment by nicoddemus** [comment]
Oh I just noticed that this Travis jobs is posting the results, but that's because I pushed to `pytest-dev/plugincompat`, which has the credentials, instead to my fork.

I'm closing this to test this hypothesis.

File: .travis.yml (hunk 1 above)
Comment (nicoddemus) [reply .travis.yml]: They are available for PRs in the main repository, but not on forks. #29 failed because of that. 😉

Comment (wimglenn) [reply .travis.yml]: Ah, OK!

**Pull request closed without merging**